--- a/macros/mate-autogen	2011-12-08 13:38:26.000000000 -0600
+++ b/macros/mate-autogen	2012-01-30 18:59:11.077748700 -0600
@@ -310,7 +310,9 @@ done
 #tell Mandrake autoconf wrapper we want autoconf 2.5x, not 2.13
 WANT_AUTOCONF_2_5=1
 export WANT_AUTOCONF_2_5
-version_check autoconf AUTOCONF 'autoconf2.50 autoconf autoconf-2.53' $REQUIRED_AUTOCONF_VERSION \
+#tell Gentoo autoconf wrapper we want autoconf 2.5x, not 2.13
+export WANT_AUTOCONF=2.5
+version_check autoconf AUTOCONF 'autoconf autoconf-2.5x' $REQUIRED_AUTOCONF_VERSION \
     "http://ftp.gnu.org/pub/gnu/autoconf/autoconf-$REQUIRED_AUTOCONF_VERSION.tar.gz"
 AUTOHEADER=`echo $AUTOCONF | sed s/autoconf/autoheader/`
 
