--- origsrc/mate-common-1.6.1/macros/mate-autogen	2013-08-06 13:52:26.796688200 -0500
+++ src/mate-common-1.6.1/macros/mate-autogen	2013-08-06 13:53:01.684683600 -0500
@@ -343,6 +343,7 @@ case $REQUIRED_AUTOMAKE_VERSION in
     1.11*) automake_progs="automake-1.13 automake-1.12 automake-1.11" ;;
     1.12*) automake_progs="automake-1.13 automake-1.12" ;;
     1.13*) automake_progs="automake-1.13" ;;
+    1.14*) automake_progs="automake-1.14" ;;
 esac
 version_check automake AUTOMAKE "$automake_progs" $REQUIRED_AUTOMAKE_VERSION \
     "http://ftp.gnu.org/pub/gnu/automake/automake-$REQUIRED_AUTOMAKE_VERSION.tar.gz"
